ui/imports/shared/controls/TokenDelegate.qml

```diff
@@ -17,16 +17,16 @@ StatusListItem {
     property alias change24HourPercentage: change24HourPercentageText
 
     property string currentCurrencySymbol
-    property string textColor: changePct24hour === undefined  ? Theme.palette.baseColor1 :
-                               Math.sign(changePct24hour) === 0 ? Theme.palette.baseColor1 :
-                               Math.sign(changePct24hour) === -1 ? Theme.palette.dangerColor1 :
+    property string textColor: modelData.changePct24hour === undefined  ? Theme.palette.baseColor1 :
+                               Math.sign(modelData.changePct24hour) === 0 ? Theme.palette.baseColor1 :
```

Comment:
`Math.sign()` not needed; I have a déjàvu ;)

Reply:
hahhaha me too now that you pointed it out!